core/templates/dev/head/pages/community-dashboard-page/contributions-and-review/contributions-and-review.directive.ts

```diff
@@ -151,61 +303,96 @@ angular.module('oppia').directive('contributionsAndReview', [
+                    getQuestionContributionsSummary());
                   ctrl.contributionsDataLoading = false;
                 });
             }
+            if (suggestionType === ctrl.SUGGESTION_TYPE_TRANSLATE) {
+              ContributionAndReviewService
+                .getUserCreatedTranslationSuggestions(
+                  function(suggestionIdToSuggestions) {
+                    ctrl.activeContributionTab = ctrl.SUGGESTION_TYPE_TRANSLATE;
+                    ctrl.contributions = suggestionIdToSuggestions;
+                    ctrl.contributionSummaries = (
+                      getTranslationContributionsSummary());
+                    ctrl.contributionsDataLoading = false;
+                  });
+            }
           };
 
-          ctrl.switchToReviewTab = function() {
-            if (!ctrl.reviewTabActive) {
-              ctrl.reviewTabActive = true;
-              ctrl.contributionsDataLoading = true;
-              ctrl.contributionSummaries = [];
-              ContributionAndReviewService.getReviewableTranslationSuggestions(
+          ctrl.switchToReviewTab = function(suggestionType) {
+            ctrl.activeContributionTab = '';
+            ctrl.contributionsDataLoading = true;
+            ctrl.contributionSummaries = [];
+
+            if (suggestionType === ctrl.SUGGESTION_TYPE_QUESTION) {
+              ContributionAndReviewService.getReviewableQuestionSuggestions(
                 function(suggestionIdToSuggestions) {
+                  ctrl.activeReviewTab = ctrl.SUGGESTION_TYPE_QUESTION;
                   ctrl.contributions = suggestionIdToSuggestions;
                   ctrl.contributionSummaries = (
-                    getTranslationContributionsSummary());
+                    getQuestionContributionsSummary());
                   ctrl.contributionsDataLoading = false;
                 });
             }
+            if (suggestionType === ctrl.SUGGESTION_TYPE_TRANSLATE) {
+              ContributionAndReviewService
+                .getReviewableTranslationSuggestions(
+                  function(suggestionIdToSuggestions) {
+                    ctrl.activeReviewTab = ctrl.SUGGESTION_TYPE_TRANSLATE;
+                    ctrl.contributions = suggestionIdToSuggestions;
+                    ctrl.contributionSummaries = (
+                      getTranslationContributionsSummary());
+                    ctrl.contributionsDataLoading = false;
+                  });
+            }
           };
 
           UserService.getUserInfoAsync().then(function(userInfo) {
             ctrl.isAdmin = userInfo.isAdmin();
             ctrl.userIsLoggedIn = userInfo.isLoggedIn();
-            ctrl.userDeatilsLoading = false;
-            username = userInfo.getUsername();
+            ctrl.userDetailsLoading = false;
             if (ctrl.isAdmin) {
-              ctrl.reviewTabActive = false;
-              ctrl.switchToReviewTab();
+              ctrl.switchToReviewTab(ctrl.SUGGESTION_TYPE_QUESTION);
```

Comment:
Any reason why ctrl.SUGGESTION_TYPE_QUESTION is hardcoded? It might be possible that the default page should be other tabs right?

Reply:
Ah I see. Should this in the long run be a setting in the preferences page? (Like how we have an option for whether the user whats to load to the learner dashboard or the creator dashboard on login).

Reply:
I think it's more just a matter of defaulting to the first item in the sidebar list (alphabetically).